src/Backend/Core/Installer/ModuleUninstaller.php

```diff
@@ -11,8 +11,8 @@
 
 /**
  * The base-class for the uninstaller
+ * @deprecated
  */
-abstract class ModuleUninstaller extends ModuleInstaller
+abstract class ModuleUninstaller extends AbstractModuleUninstaller
```

Comment:
You don't need this class at all since it is a new class in this pr so you can remove this one